zppy/templates/e3sm_to_cmip.bash

```diff
@@ -0,0 +1,91 @@
+#!/bin/bash
+{% include 'inclusions/slurm_header.bash' %}
+{% include 'inclusions/boilerplate.bash' %}
+{{ environment_commands }}
+
+# Create temporary workdir
+hash=`mktemp --dry-run -d XXXX`
+workdir=tmp.{{ prefix }}.${id}.${hash}
+mkdir ${workdir}
+cd ${workdir}
+
+tmp_dir=tmp_{{ prefix }}
+
+# Generate CMIP ts
+cat > default_metadata.json << EOF
+{% include cmip_metadata %}
+EOF
+{
+  export cmortables_dir={{ cmor_tables_prefix }}/cmip6-cmor-tables/Tables
```

Comment:
@forsyth2 I just found a bug in e3sm_to_cmip task when testing zppy (in e3sm-unified v1.10) on a Livermore machine . The cmortables_dir should be `cmortables_dir={{ cmor_tables_prefix }}/e3sm_to_cmip_data/cmip6-cmor-tables/Tables`. The tests on `LCRC` won't catch this problem, because somehow there is an excessive `cmip6-cmor-tables` directory under `/lcrc/group/e3sm/diagnostics/` which does not exist in the centralized input data locawtion.

Reply:
Thanks, fixed in https://github.com/E3SM-Project/zppy/pull/650/commits/e7d43e157e8dc45ec7ff12dd57340c0259b9f852

Reply:
thank you!